src/main/java/com/codereview/ipcounter/service/impl/FileNameArgumentProcessor.java

```diff
@@ -0,0 +1,31 @@
+package com.codereview.ipcounter.service.impl;
+
+import static com.codereview.ipcounter.utils.Constants.FILE_NAME_ARG;
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+import com.codereview.ipcounter.service.ArgumentProcessor;
+import com.codereview.ipcounter.service.FileDataProcessor;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class FileNameArgumentProcessor implements ArgumentProcessor {
+
+  private final FileDataProcessor fileDataProcessor;
+
```

Comment:
In general, there is strong code cohesion. It is very bad. Main logic should be in Handler where
first step: check args
second: open file
third: calculate
forth: show result.
And each part should be independent.